prowler/config/config.py

```diff
@@ -187,7 +188,7 @@ def load_and_validate_config_file(provider: str, config_file_path: str) -> dict:
             # and a new format with a key for each provider to include their configuration values within.
             if any(
                 key in config_file
-                for key in ["aws", "gcp", "azure", "kubernetes", "m365"]
+                for key in ["aws", "gcp", "azure", "kubernetes", "m365", "alibabacloud"]
```

Comment:
```suggestion
                for key in ["aws", "gcp", "azure", "kubernetes", "m365"]
```
I think this is not needed since it was for the legacy config. Probably m365 shouldn't necessarily be there neither.

Reply:
Done